app/views/games/categories/show.slim

```diff
@@ -1,20 +1,54 @@
 - title @game.name
 - content_for(:header) do
-  ol.breadcrumb
+  ol.breadcrumb.shadow
+    li.breadcrumb-item = link_to "Splits I/O", root_path
     li.breadcrumb-item = link_to "Games", games_path
     li.breadcrumb-item = link_to @category.game, game_path(@category.game)
     - unless @on_game_page
       li.breadcrumb-item = link_to @category, game_category_path(@game, @category)
-  h3 = @category.game
+  h1 = @category.game
+  - if @game.aliases.where.not(name: @game.name).present?
+    h6 Also known as:
+    ul
+      - @game.aliases.where.not(name: @game.name).each do |game_alias|
+        li = game_alias
   h6
     ul.run-options
+      - if @game.srl_uri.present?
+        li = link_to image_tag(asset_path("srl.png")), @game.srl_uri
```

Comment:
With all other buttons having outlines, it might be nice to have this outlined as well to maintain a consistent look, but it's completely subjective on what you like best.

Reply:
Hmm, good point but in trying to outline it the green outline actually clashes pretty bad with the SRL logo colors. I'll use a `.btn.btn-dark` to hit halfway between consistent and decent-looking.